files/usr/share/cinnamon/applets/grouped-window-list@cinnamon.org/menus.js

```diff
@@ -1175,7 +1152,6 @@ class AppThumbnailHoverMenu extends PopupMenu.PopupMenu {
                 this.appThumbnails[i].thumbnailActor.realize();
```

Comment:
This realize() call was a work-around for `setStyleOptions`, and could be removed. 

Thumbnail scaling is incorrect now - if you open enough thumbnails they clip the monitor area, probably will need to check padding in `refreshThumbnails`. That will result in more `get_length` calls, unless we figure out a way to do it once and not in every thumbnail.